### DISCUSSION TITLE
SNCF - Having Departure time instead of "in"

### DISCUSSION THREAD

**Discussion by bpasek** [yyrkoon94/lovelace-idf-mobilite#16] [Ideas]
Hello,
As I use a lot of SNCF trains, we often talks with train departure time instead of train in XX minutes.
Do you think is it possible to have something that looks like new screens?
https://pbs.twimg.com/media/FyqtqcUaIAAcQtU?format=jpg&name=900x900
I know that's a complete new design

Thanks a lot,

Brice

**Comment by yyrkoon94** [comment]
Hello @bpasek,

I can probably create a specific design that can be activated with a boolean like "Display by departure time".
To achieve this, can you share your configuration (MonitoringRef and eventually LineRef) to have a specific exemple to work on ?

**Reply by bpasek** [reply]
Hello, sure, MonitoringRef: "STIF:StopArea:SP:64145:"

Thanks a lot !! :)

**Comment by stivxgamer** [comment]
je préférerais également avoir l'heure de départ de mon train. Mais c'est déjà du beau travail. un grand merci !

**Comment by yyrkoon94** [comment]
New version available with hour instead of delay and platform name
